README.md

```diff
@@ -49,12 +49,15 @@ You request access to the anonymised toy dataset from
 
 ## Required packages
 * R (>= 3.1.0),
-* XML,
 * data.table,
-* yaml,
+* dbplyr,
+* dplyr,
+* methods,
 * pander,
 * Rcpp,
-* methods
+* RPostgreSQL,
+* XML,
```

Comment:
Presumably once the DB connector is working we could strip out the old XML handling code?

But not in this PR!

Reply:
I don't know about that. Based on Sinan's messages about the package being general purpose, keeping the XML handling seems like it could be sensible since it may not be used just for the database we have?

Reply:
I'll ask this morning :)